### PR TITLE
Ds 3202

### PR DESCRIPTION
**Pull request by chschan**

### CODE REVIEW

File: R/chartutils.R [hunk 1]
```diff
@@ -102,17 +103,24 @@ checkMatrixNames <- function(x, assign.col.names = TRUE)
         dimnames(new.x) <- old.names
     }
 
-    # Many of the charts use the statistic attribute
-    x <- CopyAttributes(new.x, x)
+    # Convert percentage data to decimal form
+    stat <- attr(x, "statistic")
```
Comment (JustinCCYap) [inline R/chartutils.R:107]: Why is this line repeated (line 93)

File: R/chartutils.R [hunk 2]
```diff
@@ -102,17 +103,24 @@ checkMatrixNames <- function(x, assign.col.names = TRUE)
...
+    if (is.null(stat) && !is.null(dimnames(x)))
+        stat <- dimnames(x)[[length(dim(x))]][1]
```
Comment (JustinCCYap) [inline R/chartutils.R:109]: Does the final dimension always contain the statistics?

File: R/labeledscatterchart.R [hunk 3]
```diff
@@ -173,6 +173,13 @@ LabeledScatter <- function(x = NULL,
             logo.urls <- rep(logo.urls, num.tables)
     }
 
+    tmp.stat <- attr(x, "statistic")
+    if ((is.array(x) || is.numeric(x)) && isTRUE(grepl("%", tmp.stat)))
+    {
+        x <- x/100
+        attr(x, "statistic") <- NULL
```
Comment (JustinCCYap) [inline R/labeledscatterchart.R:180]: Is the data always to be shown as a proportion?

File: R/chartutils.R (hunk 2 above)
Comment (chschan) [reply R/chartutils.R:108]: Yep, if the statistic attribute is NULL, then its always in the final dimension

File: R/labeledscatterchart.R (hunk 3 above)
Comment (chschan) [reply R/labeledscatterchart.R:180]: For all the plotly charts, percentage data is always expected to be entered as a proportion. We sent `xxx.format` (via PrepareNumbers) to indicate that the axis tick/hovertext/data labels should be shown as a percentage.

File: R/chartutils.R (hunk 1 above)
Comment (chschan) [reply R/chartutils.R:106]: Thanks for picking that up. I've removed the repeated line in 47c0894

**Pull request merged**